.gitmodules

```diff
@@ -54,3 +54,6 @@
 [submodule "src/examples/gyro_fft/CMSIS_5"]
 	path = src/examples/gyro_fft/CMSIS_5
 	url = https://github.com/ARM-software/CMSIS_5.git
+[submodule "src/lib/events/libevents"]
+	path = src/lib/events/libevents
+	url = https://github.com/bkueng/libevents.git
```

Comment:
What was the reasoning for having this as a separate repo? Stability? Would this make sense to move under `PX4/libevents` once this PR is merged?

Reply:
No, there are several things (common event defs, scripts, code generation) that is shared with other projects (QGC, log analysis, possibly MAVSDK in future).